active/NavigationView/HierarchicalNav/HierarchicalNavigationView.md

```diff
@@ -0,0 +1,265 @@
+
+Selected items will draw their selection indicators along their left edge when in left mode or their bottom edge when in top mode. 
+The selected item may not always remain visible.
+For example, the selected item may be a child node inside a non-expanded subtree.
+In this situation, the first visible ancestor of the selected item will show as selected, and the selection indicator will move as users expand the subtree. 
+The entire navigation view will show no more than one selection indicator.
+
+## Keyboarding
+Users can move focus around the navigation view using their [keyboard](https://docs.microsoft.com/en-us/windows/uwp/design/input/keyboard-interactions). 
+The arrow keys expose "inner navigation" within the pane and follow the interactions provided in [tree view](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/tree-view).
+- **Up arrow** moves focus to the item directly above the item currently in focus
+- **Down arrow** moves focus the item directly below the item currently in focus; note that the items do not need to be visually adjecent, focus will move from the last item in the pane's list to the settings item.
+- **Right arrow**
+  - if the current item is a collapsed parent with children, right arrow expands the item but does not move focus
+  - if the current item is an expanded parent with children, right arrow moves focus to the first child
+  - if the current item is a leaf node, right arrow does nothing
+- **Left arrow** 
+  - if the current item is a collapsed parent with children, left arrow does nothing
+  - if the current item is an expanded parent with children, left arrow collapse the item but does not move focus
+  - if the current item is a leaf node, left arrow moves focus to the current item's parent
+
+# Examples
+
+## Add items in markup
+Declare app navigation hierarchy in markup.
+
+```Xaml
+<muxc:NavigationView>
+    <muxc:NavigationView.MenuItems>
+        <muxc:NavigationViewItem Content="Home" Icon="Home" ToolTipService.ToolTip="Home"/>
+        <muxc:NavigationViewItem Content="Collections" Icon="Keyboard" ToolTipService.ToolTip="Collections">
+            <muxc:NavigationViewItem.MenuItems>
+                <muxc:NavigationViewItem Content="Bookshelf" Icon="Library" 
+                    ToolTipService.ToolTip="Bookshelf"/>
+                <muxc:NavigationViewItem Content="Mail" Icon="Mail" ToolTipService.ToolTip="Mail"/>
+            </muxc:NavigationViewItem.MenuItems>
+        </muxc:NavigationViewItem>
+    </muxc:NavigationView.MenuItems>
+</muxc:NavigationView>
+```
+
+## Add items using data binding
+
+```xaml
+<DataTemplate x:Key="NavigationViewMenuItem" x:DataType="local:Category">
+    <muxc:NavigationViewItem Content="{x:Bind Name}" MenuItemsSource="{x:Bind Children}" 
+        SelectsOnInvoked="{x:Bind IsLeaf}"/>
```

Comment:
>IsLeaf [](start = 34, length = 6)

Do we require IsLeaf on the data object in order to support this ?

Reply:
I think this was taken from the test page of my PR. This is not required.

Reply:
Dilip is correct, that's exactly where I copied the code from. I'll remove it from here and instead add a sample showing how to use SelectsOnInvoked.